src/components/modals/create-user.jsx

```diff
@@ -17,7 +17,8 @@ class CreateUser extends Component {
       name: '',
       email: '',
       phoneNumber: '',
-      password: ''
+      password: '',
+      role: ''
```

Comment:
default this to student
```suggestion
      role: 'student'
```